phpunit.xml.dist

```diff
@@ -70,7 +70,6 @@
     </extensions>
 
     <php>
-        <env name="XDEBUG_MODE" value="coverage"/>
```

Comment:
Care to explain? I think this is required for code coverage to work. Typically a developer would set this in their environment but I think setting it here covers Actions and other CI pipelines as well.

Reply:
It does nothing. It is too late to set the environment variable.
See https://github.com/sebastianbergmann/php-code-coverage/issues/834#issuecomment-736324897

And you can see Coveralls got the coverage on this PR.